README.md

```diff
@@ -481,3 +481,70 @@ that this is expected behavior by adding this library's extension in your `phpst
 includes:
   - vendor/square/pjson/extension.neon
 ```
+
+## Laravel Integration
+
+### via castable
+
+If you wish to cast Eloquent model attributes to classes via Pjson, you might do so with the provided casting utilities:
+
+```php
+use Illuminate\Contracts\Database\Eloquent\Castable;
+
+class Schedule implements Castable // implement the laravel interface
+{
+    use JsonSerialize;
+    use JsonCastable; // use the provided Pjson trait
```

Comment:
I think it's worthwhile to include the `use` statements for these traits as well.